hoomd/hpmc/update.py

```diff
@@ -631,67 +506,31 @@ def _attach(self):
 
         if not integrator._attached:
             raise RuntimeError("Integrator is not attached yet.")
-        self._cpp_obj = cpp_cls(self._simulation.state._cpp_sys_def,
-                                integrator._cpp_obj,
-                                int(self.seed))
-        super()._attach()
-
-    @property
-    def counter(self):
-        """Get the number of accepted and rejected cluster moves.
 
-        Returns:
-            A counter object with pivot, reflection, and swap properties. Each
-            property is a list of accepted moves and rejected moves since the
-            last run.
-
-        Note:
-            `None` when the simulation run has not started.
-        """
-        if not self._attached:
-            return None
+        if isinstance(self._simulation.device, hoomd.device.GPU):
+            self._cpp_obj = cpp_cls(self._simulation.state._cpp_sys_def,
+                                    integrator._cpp_obj,
+                                    int(self.seed))
         else:
-            return self._cpp_obj.getCounters(1)
-
-    @log(category='sequence')
-    def pivot_moves(self):
```

Comment:
Why did we remove all move counters from the user facing API?

Reply:
See above, since update.clusters() is now rejection-free there is no point in reporting an acceptance rate